ui/custom_standart_item_model.py

```diff
@@ -128,10 +129,9 @@ def populate_with_data(self, data, view_name: str) -> None:
                 )
                 parent_path = node_data.parent_path
                 parent_item = self.get_item_by_path(parent_path)
-                if parent_item or parent_path == "/":
```

Comment:
I'm not entirely sure if it is save to remove this, but all unittests and the manual testing went fine.
This allows the Hdas to be created without a parent.